src/ConnectedMode/Migration/IUnintrusiveBindingController.cs

```diff
@@ -18,18 +18,18 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
+using System;
 using System.ComponentModel.Composition;
 using System.Threading;
 using SonarLint.VisualStudio.ConnectedMode.Binding;
 using SonarLint.VisualStudio.Core.Binding;
-
 using Task = System.Threading.Tasks.Task;
 
 namespace SonarLint.VisualStudio.ConnectedMode.Migration
 {
     internal interface IUnintrusiveBindingController
     {
-        Task BindAsync(BoundSonarQubeProject project, CancellationToken token);
+        Task BindAsync(BoundSonarQubeProject project, IProgress<MigrationProgress> progress, CancellationToken token);
```

Comment:
Actually, this doesn't look quite right. The `UnintrusiveBindingController` is the general-purpose-update-silently-in-the-background that will be used for updating QP unobtrusively
i.e. it is not migration-specific, so it shouldn't know about `IMigrationProcess`.

Wouldn't it be better if:
* the `UnintrusiveBindingController` used the `FixedStepsProgress`
* the `ConnectedModeMigration` class created the adapter and passed it to the `UnobtrusiveBindingController` ?

Reply:
Yea you are right, I think I reasoned that it is not all purpose because it is in the migration folder however that is bad logic. I'll change it :)

Reply:
> I think I reasoned that it is not all purpose because it is in the migration folder

Good point. It should be in the `Binding` folder instead. I'll add a note to the board to remind us to move it.